packages/core/admin/server/src/domain/condition/index.js

```diff
@@ -19,7 +19,7 @@ const { pipe, merge, set, pick } = require('lodash/fp');
 
 /**
  * Set of attributes used to create a new {@link Action} object
- * @typedef {Condition, { uid: string }} CreateConditionPayload
+ * typedef {Condition, { uid: string }} CreateConditionPayload
```

Comment:
What's the reason for this change? will it be problematic to have jsdoc types during the migration?

Reply:
I think not, for some reason the Condition type definition was colliding with another internal Typescript definition, and was getting a BLOCK-SCOPED error.